.buildkite/pipeline.full.yml

```diff
@@ -604,6 +604,7 @@ steps:
     agents:
       queue: windows-unity-wsl
     env:
+      UNITY_DIR: "/mnt/f/unity/2018.4.36f1"
       UNITY_VERSION: "2018.4.36f1"
```

Comment:
Do we really need to provide both?  I wouldn't be too surprised if we did, but it feels messy to have to.  But equally it's not really the remit of this PR to straighten it out - if both are needed then perhaps you could just raise a ticket to review the variables we have and consolidate/rationslise in some way?

Reply:
Im going to raise a separate ticket for this so that I can allocate some proper time to think of a better solution.